platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -332,17 +323,14 @@ sep10:
   ## This is the timeout period after the client has authenticated.
   #
   jwt_timeout: 86400
-  ## @param: omnibus_account_list
-  ## A comma-separated list of omnibus accounts (comma separated).
-  ## The SEP-10 omnibus account is described at:
-  ##     https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#memos
-  #
-  omnibus_account_list:
-  ## @param: require_known_omnibus_account
+  ## @param: require_known_custodial_account
   ## @type: bool
-  ## Whether to require authenticating clients to be in the omnibusAccountList
+  ## Whether to require authenticating clients to be in the list of known custodial accounts.
+  #
+  # If the flag is set to true, the client must be one of the custodial clients defined in the clients section
+  # of this configuration file.
   #
-  require_known_omnibus_account: false
+  require_known_custodial_account: false
```

Comment:
This is a breaking change, right?

Reply:
Yes... I assume the impact would be minimum if no anchors on version `2.x` are using this flag. 

WDYT?

Reply:
Don't think any custodial wallets have integrated with any of the SEP-24 anchors live on the platform yet, so we may be ok to make this change.

Reply:
nit: can we match the phrasing of `client_attribution_required` and use `known_custodial_account_required`?

Reply:
`custodial_account_required`? WDYT?